src/TextLogging/include/BipedalLocomotion/TextLogging/Logger.h

```diff
@@ -8,6 +8,7 @@
 #ifndef BIPEDAL_LOCOMOTION_TEXT_LOGGING_LOGGER_H
 #define BIPEDAL_LOCOMOTION_TEXT_LOGGING_LOGGER_H
 
+#include <spdlog/fmt/ostr.h>
```

Comment:
what is this include for?

Reply:
Required to print Eigen::Vectors (https://github.com/gabime/spdlog/issues/1638)

Reply:
Lovely!